README.md

```diff
@@ -38,14 +38,21 @@ if let tcxData = tcxData {
 
 ### Encode
 
-```swift
+Each Trackpoint time has to be unique. A per second increment from the previous. Below example uses workoutStartDateTime as placeholder. Change as appropriate on your codebase.
 
+```swift
+let workoutStartDateTime = Date()
 let build = Build(version: Version(major: 0, minor: 1, buildMajor: 0, buildMinor: 0), time: nil, builder: nil, type: .alpha)
 let author = Author(name: "TcxDataProtocol", build: build, language: nil, partNumber: "11-22-33")
+let track = [Track(trackPoint: [
+                    Trackpoint(time: workoutStartDateTime, position: nil, altitude: nil, distance: 1.11, heartRate: HeartRateInBeatsPerMinute(heartRate: 100), cadence: 100, sensorState: SensorState.present, extensions: [Extension(activityTrackpointExtension: ActivityTrackpointExtension.init(speed: 11.2, runCadence: nil, watts: 111, cadenceSensor: CadenceSensorType(rawValue: "bike")), activityLapExtension: nil, activityGoals: nil)])
+                      ,
+                    Trackpoint(time: workoutStartDateTime, position: nil, altitude: nil, distance: 1.11, heartRate: HeartRateInBeatsPerMinute(heartRate: 101), cadence: 101, sensorState: SensorState.present, extensions: [Extension(activityTrackpointExtension: ActivityTrackpointExtension.init(speed: 11.3, runCadence: nil, watts: 222, cadenceSensor: CadenceSensorType(rawValue: "bike")), activityLapExtension: nil, activityGoals: nil)])
```

Comment:
if you make mention that each Trackpoint needs an increment that should be shown here.. right now this is no change

Reply:
I made the new changes based on what I am using after I figured out how to loop the data. (I'm not sure how to show the changes like you did from the above tho)